accounting/invoice/utils.py

```diff
@@ -85,22 +106,27 @@ def send_email_with_invoice(template, subject, extra_email_context=None, client_
     template = get_template(template)
     for client_username in client_usernames:
         client = Account.objects.get(user__username=client_username)
-        providers = [rate.provider for rate in client.client_hourly_rates.all()]
+        providers = [rate.provider for rate in client.client_hourly_rates.all() if rate.active]
```

Comment:
@UmanShahzad Same comment here about using `filter` instead of `all`.

Reply:
:+1: